docs/references/bib.bib

```diff
@@ -181,6 +181,20 @@ @article{vigna:2014
 	year = {2014},
 }
 
+@article{vose:1991,
+	abstract = {Let ξ be a random variable over a finite set with an arbitrary probability distribution. Improvements to a fast method of generating sample values for ξ in constant time are suggested.},
```

Comment:
Can you use `\\( \\)` TeX markup, similar to how we do in JSDoc? See also the bib entry which follows.